godot-codegen/src/class_generator.rs

```diff
@@ -1144,6 +1144,9 @@ fn make_function_definition(sig: &FnSignature, code: &FnCode, ctx: &mut Context)
     let is_varcall = code.variant_ffi.is_some();
     let [params, variant_types, arg_exprs, arg_names] =
         make_params(sig.method_args, is_varcall, ctx);
+    let args_indices: Vec<Literal> = (0..arg_exprs.len())
+        .map(Literal::usize_unsuffixed)
+        .collect();
```

Comment:
The `collect()` might not be necessary, as `quote!` can consume iterators -> you would save an allocation.

Sometimes it is needed because the iterator keeps borrows of other local variables, and collecting breaks that data dependency.